dist/workflow_step_template_schema.json

```diff
@@ -858,7 +843,7 @@
                 "1.0.0",
                 "1.0.1"
             ],
-            "default": "1.0.0"
+            "default": "1.0.1"
```

Comment:
My bad, I did not regenerate the schema in the related PR. Let's update the PR template

```md
- [ ] CHANGELOG entry is correct
- [ ] I ran the build command after making changes to the schema
- [ ] I manually tested the changes and checked that related features do not have any regressions
```